packages/core-blockchain/package.json

```diff
@@ -25,7 +25,8 @@
     "async": "^2.6.1",
     "delay": "^2.0.0",
     "interval-to-human": "^0.1.1",
-    "xstate": "^3.2.1"
+    "xstate": "^3.2.1",
+    "pretty-ms": "^3.2.0"
```

Comment:
This is not properly installed. It needs to be removed and then added with `lerna add pretty-ms --scope=@arkecosystem/core-blockchain`.

Reply:
ty for this insight